sdk/attestation/Azure.Security.Attestation/src/AttestationClient.cs

```diff
@@ -82,64 +84,76 @@ protected AttestationClient()
         /// <summary>
         /// Attest an Intel SGX enclave.
         /// </summary>
-        /// <param name="quote">An Intel SGX "quote".
-        /// See https://software.intel.com/content/www/us/en/develop/articles/code-sample-intel-software-guard-extensions-remote-attestation-end-to-end-example.html" for more information.</param>
-        /// <param name="initTimeData">Data provided when the enclave was created.</param>
-        /// <param name="initTimeDataIsObject">true if the initTimeData parameter should be treated as an object, false if it should be treated as binary.</param>
-        /// <param name="runTimeData">Data provided when the quote was generated.</param>
-        /// <param name="runTimeDataIsObject">true if the runTimeData parameter should be treated as an object, false if it should be treated as binary.</param>
+        /// <param name="request">Aggregate type containing the information needed to perform an attestation operation.</param>
         /// <param name="cancellationToken">Cancellation token used to cancel the request.</param>
-        /// <returns>An <see cref="AttestationResponse{AttestationResult}"/> which contains the validated claims for the supplied <paramref name="quote"/>, <paramref name="runTimeData"/>, and <paramref name="initTimeData"/></returns>
-            public virtual AttestationResponse<AttestationResult> AttestSgxEnclave(ReadOnlyMemory<byte> quote, BinaryData initTimeData, bool initTimeDataIsObject, BinaryData runTimeData, bool runTimeDataIsObject, CancellationToken cancellationToken = default)
-                => AttestSgxEnclaveInternal(quote, initTimeData, initTimeDataIsObject, runTimeData, runTimeDataIsObject, false, cancellationToken).EnsureCompleted();
+        /// <returns>An <see cref="AttestationResponse{AttestationResult}"/> which contains the validated claims for the supplied <paramref name="request"/>.</returns>
+        /// <remarks>The <see cref="AttestationRequest.Evidence"/> must be an Intel SGX Quote.
+        /// <seealso href="https://software.intel.com/content/www/us/en/develop/articles/code-sample-intel-software-guard-extensions-remote-attestation-end-to-end-example.html"/>  for more information.
```

Comment:
I'm reaching out to see if I can find a language neutral page. Unfortunately, since this is a 3rd party page, I'm not 100% sure that there is one available (this is especially true since Intel has shifted their SGX client work to OpenEnclave).

Reply:
Unfortunately there isn't a language neutral page. There's an english page, and a chinese page, but both require the language tag.